README.md

```diff
@@ -17,8 +17,42 @@ everything feasible.
 
 ## Testing
 
-**Note:** The tests currently expect a [SaltyRTC Server][server] instance to
-run on `localhost:6699`.
+### Setup
+
+The integration tests currently expect a [SaltyRTC Server][server] instance to
+run on `localhost:8765`.
+
+First, create a test certificate for localhost, valid for 5 years.
```

Comment:
I don't think the validity matters here. :)

Reply:
Removed